tests/gui/builder/control/test_slider.py

```diff
@@ -36,6 +36,20 @@ def test_slider_with_min_max_builder(gui: Gui, test_client, helpers):
     helpers.test_control_builder(gui, page, expected_list)
 
 
+def test_slider_with_step_builder(gui: Gui, test_client, helpers):
+    gui._bind_var_val("x", 0)
+    with tgb.Page(frame=None) as page:
+        tgb.slider(value="{x}", min=-10, max=10, step=2)
+    expected_list = [
+        "<Slider",
+        "min={-10.0}",
+        "max={10.0}",
+        "step={2.0}",
```

Comment:
the update to factory.py should fix this
PS: You also need to add jsx test ;-)

Reply:
Thanks for guiding, @FredLL-Avaiga . 

I fixed what you commented on. I'm not sure about jsx test. Do I just need to add a similar function to the `tests/gui/control/test_slider.py` or something else?

Reply:
you can add a test in Slider.spec.tsx if it makes sense
PS: I've updated your branch

Reply:
Yeah, it makes sense. I'm unfamiliar with testing but I will have a look at it later and let you know how it goes.

Reply:
testing the behavior of the component needs to be done in the Slider.spec.tsx

Reply:
I added a test to `Slider.spec.tsx`,  and used fireEvent for that. 

I tried to trigger slider one increment/decrement by function but couldn't find the way so. I just did like
```
const step = parseFloat(elt.step);
fireEvent.change(elt, {target:{value: 33 + step }});
```

Reply:
Can you try `await user.keyboard('[ArrowRight]')` after `userEvent.click(<the right element>)` 
And then check the value ?

Reply:
```
it("changes by set step", async()=>{
        const dispatch = jest.fn();
        const state: TaipyState = INITIAL_STATE;
        const { getByDisplayValue } = render(
            <TaipyContext.Provider value={{ state, dispatch }}>
                <Slider value={33} min={1} max={50} step={0.1} />
            </TaipyContext.Provider>
        );
        const elt = getByDisplayValue("33") as HTMLInputElement;
        await userEvent.click(elt);
        await userEvent.keyboard('[ArrowRight]')
        expect(elt.value).toBe("33.1");
        });
```
I tried like this. The click itself changes the value to 1 (min value) from 33, but the keyboard doesn't change the value at all.

Reply:
Sounds good to me. I just removed the change on the Slider.spec.tsx and added a property explanation to the viselements.json.

Reply:
@FredLL-Avaiga  Is it fine to make this PR ready for review?

Reply:
Yes I think so